docs/configuration.md

```diff
@@ -45,7 +53,9 @@ public class Example {
 3. Load configuration from an [`InputStream`](https://docs.oracle.com/javase/8/docs/api/java/io/InputStream.html)
 4. Load configuration from the Symphony directory. The Symphony directory is located under your `${user.home}/.symphony` 
     folder. It can be useful when you don't want to share your own Symphony credentials within your project codebase
-5. Last but not least, you can obviously define your configuration object as a [POJO](https://en.wikipedia.org/wiki/Plain_old_Java_object) 
+5. Load configuration from [`Map`](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html) of properties
+6. Load configuration from [`Properties`](https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html)
```

Comment:
Please update to java 8 documentation to stay consistent: https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html